hadoopsubmarine/submarine
# Add a Dockerfile and the related scripts for submarine dev and beginner

### What is this PR for?
To avoid the environment setup burden for the submarine beginners and developer, I propose a all-in-one docker image. The images containers hdfs, yarn, spark, submarine. One command for bootstrap the daemons to run a single node hadoop and one command for running a submarine/spark job.

See README.txt for more details.


### What type of PR is it?
Feature

### Todos
One document

### How should this be tested?
Not sure if Travis CI can build the image and run the samples in it to verify. https://submarine.apache.org/contribution/contributions.html#continuous-integration

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes
